collections/_article/dev-snapshot-godot-4-4-beta-1.md

```diff
@@ -0,0 +1,289 @@
+---
+title: "Dev snapshot: Godot 4.4 beta 1"
+excerpt: "Godot 4.4 "
```

Comment:
TODO

Reply:
Nevermind this stale suggestion, the TODO was resolved already.